packages/@aws-cdk/aws-codebuild/test/test.project.ts

```diff
@@ -1466,6 +1466,124 @@ export = {
 
         test.done();
       },
+
+      'can be provided as a SecretValue for SecretsManager'(test: Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
+
+        // WHEN
+        const secretValue = secretsmanager.Secret.fromSecretCompleteArn(stack, 'Secret',
+          'arn:aws:secretsmanager:us-west-2:123456789012:secret:mysecret-123456').secretValueFromJson('json-key');
```

Comment:
Can you split this into 2 expressions please? It actually took me quite a while to understand you're not passing the `Secret` here directly...

```ts
        const secret = secretsmanager.Secret.fromSecretCompleteArn(stack, 'Secret',
          'arn:aws:secretsmanager:us-west-2:123456789012:secret:mysecret-123456');
        const secretValue = secret.secretValueFromJson('json-key');
```